build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs

```diff
@@ -184,7 +184,6 @@ public CommandResult Execute()
             _process.EnableRaisingEvents = true;
 
             var sw = Stopwatch.StartNew();
-            ReportExecBegin();
```

Comment:
I think these should be left in.  Then "ReportExecEnd" has been very helpful in diagnosing build failures when these Exec commands fail.